packages/pagination/README.md

```diff
@@ -102,3 +102,46 @@ new LinkHeaderInterceptor({ resource: 'data', pageName: '_page', perPageName: 'n
 
 @MongoPaginationParamDecorator({ pageName: '_page', perPageName: 'numberPerPage', defaultLimit: 50  })
 ```
+
+You can also have the pagination in the response body by using the PaginationBodyInterceptor. By using this interceptor, the pagination is included in the response body instead of the header.
+
+A paginated data response is returned.
+
+```json 
+{
+  resources: T[];
+  pagination: {
+    next: string | null;
+    previous: string | null;
+    first: string | null;
+    last: string | null;
+    totalPages: number | null;
+    totalResources: number | null;
+  };
+}
```

Comment:
This is not a JSON 😉 

```suggestion
```json
{
        "resources": [],
        "pagination": {
          "next": null,
          "previous": null,
          "first": null,
          "last": null,
          "totalPages": 1,
          "totalResources": 0
        }
      }
```